udata/core/badges/tests/test_model.py

```diff
@@ -155,3 +156,21 @@ def test_create_disallow_unknown_badges(self):
         with self.assertRaises(db.ValidationError):
             fake = Fake.objects.create()
             fake.add_badge("unknown")
+
+
+class DatasetBadgeTest(DBTestMixin, TestCase):
```

Comment:
Do you want to move this in test_dataset_model?

Reply:
Fixed the badge tests ;) What do you think?

Reply:
:ok_hand: looks good, thank you!